internal/compiler-interface/src/main/contraband/definition.json

```diff
@@ -101,11 +101,11 @@
           "type": "record",
           "fields": [
             { "name": "definitionType",        "type": "DefinitionType" },
-            { "name": "selfType",              "type": "lazy Type"      },
-            { "name": "structure",             "type": "lazy Structure" },
+            { "name": "selfType",              "type": "Type"           },
+            { "name": "structure",             "type": "Structure"      },
             { "name": "savedAnnotations",      "type": "String*"        },
             { "name": "childrenOfSealedClass", "type": "Type*"          },
-            { "name": "topLevel",              "type": "Boolean"        },
+            { "name": "topLevel",              "type": "boolean"        },
```

Comment:
Glad you found that issue, we need to be pretty careful with equality on these fields.